open-machine-learning-jupyter-book/data-science/data-visualization/visualization-distributions.md

```diff
@@ -15,22 +15,23 @@ kernelspec:
 
 # Visualizing distributions
 
-In the previous lesson, you learned some interesting facts about a dataset about the birds of Minnesota. You found some erroneous data by visualizing outliers and looked at the differences between bird categories by their maximum length.
+In the previous section, you learned some interesting facts about a dataset about the birds of Minnesota. You found some erroneous data by visualizing outliers and looked at the differences between bird categories by their maximum length.
 
 ## Explore the birds dataset
 
 Another way to dig into data is by looking at its distribution, or how the data is organized along an axis. Perhaps, for example, you'd like to learn about the general distribution, for this dataset, of the maximum wingspan or maximum body mass for the birds of Minnesota. 
 
-Let's discover some facts about the distributions of data in this dataset. In the _notebook.ipynb_ file at the root of this lesson folder, import Pandas, Matplotlib, and your data:
+Let's discover some facts about the distributions of data in this dataset. Create a new jupyter notebook file name as `bird-distributions.ipynb`, import Pandas, Matplotlib, and your data:
```

Comment:
Maybe you can create this empty notebook file in the assignment and refer it here?